docs/pages/installation/windows.md

```diff
@@ -131,31 +136,33 @@ The Docker Desktop application for Windows includes a comprehensive set of tools
 ### 4. <a name="step4"> <b>Install a Kubernetes cluster</b></a>
 
 Once Docker is installed, the next step will be to install a Kubernetes cluster.
-Under "Settings" in the Docker Desktop application, enable *Kubernetes*.
+Under "Settings" in the Docker Desktop application, enable _Kubernetes_.
 In this how-to, [K3d](https://github.com/rancher/k3d) will be used as it relies only on Docker.
 
   <pre class="codeblock-pre">
   <div class="codeblock"><div class="clipboardjs">curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
 k3d cluster create
-export KUBECONFIG="$(k3d kubeconfig get 'k3s-default')"</div></div>
+$env:KUBECONFIG = "$(k3d.exe kubeconfig get 'k3s-default')"</div></div>
```

Comment:
Here, we are using bash script, we only needed the change in line 153, please revert this one